src/advanced-settings/setting-card/SettingCard.jsx

```diff
@@ -20,8 +20,25 @@ const SettingCard = ({
           <Card.Header
             title={displayName}
             subtitle={(
-              <OverlayTrigger placement="bottom" overlay={<Tooltip id="setting-description">{help}</Tooltip>}>
-                <Icon src={Info} />
+              <OverlayTrigger
+                trigger="click"
+                rootClose
+                placement="bottom"
+                overlay={(
+                  <Popover id="popover-positioned">
+                    <Popover.Content>
+                      {/* eslint-disable-next-line react/no-danger */}
+                      <div dangerouslySetInnerHTML={{ __html: help }} />
+                    </Popover.Content>
+                  </Popover>
+                )}
+              >
+                <IconButton
+                  src={Info}
+                  iconAs={Icon}
+                  alt="Show help text"
```

Comment:
optional: Do we need to wrap alt text into intl?

Reply:
Yes, good catch 👍

Reply:
@PKulkoRaccoonGang 
Please let me know when you wrap the `alt`, I'll then merge the PR